src/config.rs

```diff
@@ -738,4 +752,27 @@ mod tests {
         spotifyd_section.username = Some("testUserName".to_string());
         assert_eq!(merged_config, spotifyd_section);
     }
+
+    #[cfg(features = "alsa_backend")]
+    #[test]
+    fn test_section_merging() {
+        let mut spotifyd_section = SharedConfigValues::default();
+        global_section.mixer = "PCM".to_string();
+
+        let global_section = SharedConfigValues::default();
+        global_section.mixer = "PCM1".to_string();
+
+        // The test only makes sense if both sections differ.
+        assert!(spotifyd_section != global_section, true);
```

Comment:
`assert!()` tests by itself, whether the expression is true.

```suggestion
        assert!(spotifyd_section != global_section);
```